src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+html,
+body,
+ul {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
This property is not inherited and you didn't set any paddings or borders to this tags, so it's unnecessary here